app/gui/src/dashboard/layouts/Drive/Categories/Category.ts

```diff
@@ -23,15 +23,15 @@ import { newDirectoryId } from '#/services/LocalBackend'
 const PATH_SCHEMA = z.string().refine((s): s is Path => true)
 const DIRECTORY_ID_SCHEMA = z.string().refine((s): s is DirectoryId => true)
 
-const EACH_CATEGORY_SCHEMA = z.object({ label: z.string(), icon: z.string() })
+const EACH_CATEGORY_SCHEMA = z.object({
+  label: z.string(),
+  icon: z.string(),
+  homeDirectoryId: DIRECTORY_ID_SCHEMA,
```

Comment:
@MrFlashAccount thoughts on this change?

Reply:
This is misleading because recent and trash folders do not have directory id schema. 

This Might be `homeDirectoryId: DIRECTORY_ID_SCHEMA.or(z.null())` I think